frontend/src/features/public-form/components/FormFields/FormSectionsContext.tsx

```diff
@@ -47,20 +52,20 @@ export const FormSectionsProvider = ({
    * Set default active section id on first load of the form.
    */
   useEffect(() => {
-    if (isFirstLoad && orderedSectionFields) {
-      setActiveSectionId(orderedSectionFields[0]?._id)
+    if (isFirstLoad && orderedSectionFieldIds) {
+      setActiveSectionId(orderedSectionFieldIds?.[0])
```

Comment:
nit: probably don't need nullable check since truthy check is performed already
```suggestion
      setActiveSectionId(orderedSectionFieldIds[0])
```

Reply:
fixed